src/routes/sensorthings.js

```diff
@@ -0,0 +1,50 @@
+import proxy                   from 'express-http-proxy';
+
+import config                  from '../config';
+import { resourceEndpoints }   from '../routes/base';
+import sensorthings            from 'sensorthings'
+
+const localDb = config.get('sensorthings.local.db');
+let sensorthingsAPI;
+
+if (localDb) {
+  //Init DB
+  sensorthingsAPI = sensorthings;
+} else {
+  /**
+   * This is just a temporary utility.
+   *
+   * This proxy allows us to quickly expose a SensorThings API while we work on
+   * our own open source implementation of this API. We will be using SensorUp's
+   * sandbox.
+   *
+   * You will need to get an access token by creating an account at
+   * https://pg.sensorup.com/playground.html
+   *
+   * Add the access token to your 'sandboxToken' config file entry.
```

Comment:
Maybe reword this comment to explain that we use SensorUp's for now, but this code is ready to proxy to other ST external services. Also, a comment at the top of the file explaining the local vs remote choices might be helpful.